resources/public/api/conf/2.0/schemas/amend_request_schema.json

```diff
@@ -8,21 +8,6 @@
       "type":"object",
       "description":"Object holding the booleans for annual allowance.",
       "properties":{
-        "isAnnualAllowanceReduced":{
```

Comment:
It was misleading having these in the V2 OAS as it makes it sound like they can still be used, when in fact they have been removed.